src/services/FacilityService.ts

```diff
@@ -0,0 +1,24 @@
+import api from '@/api';
+
+const getNearByStores = async (payload: any): Promise <any>  => {
```

Comment:
As `storeLookup` api is used to fetch all stores and its not specific to only fetch nearBy stores
```suggestion
const getStores = async (payload: any): Promise <any>  => {
```

Reply:
Improev sir

Reply:
Improved sir